3.functions/Q3.c

```diff
@@ -5,3 +5,21 @@
     // Example:\n
     // Input: 100 C\n
     // Output: 212 F\n
+
+#include<stdio.h>
+
+float celsiusToFahrenheit(float celsius){
+    // F = (9/5)C + 32.
+    float Fahrenheit;
+    Fahrenheit = (float)(celsius * 9 / 5) + 32;
+    return Fahrenheit;
+}
+
+int main(){
+    float degreeCelsius;
```

Comment:
read question again, ask user desired conversion and write function for fahrenheitToCelsius also.